README.md

```diff
@@ -41,20 +41,20 @@ $ npm install asset-pipe-js-reader
 Make an JavaScript bundle out of two asset feeds stored on a filesystem:
 
 ```js
-const Reader = require('asset-pipe-js-reader');
-const SinkFs = require('asset-pipe-sink-fs');
+const bundleJS = require('@asset-pipe/js-reader');
+const SinkFs = require('@asset-pipe/sink-fs');
 
 const sink = new SinkFs({
     path: './assets'
 });
 
-const feedA = sink.reader('a.json');
-const feedB = sink.reader('b.json');
-
-const reader = new Reader([feedA, feedB]);
-reader.on('pipeline ready', () => {
-    reader.pipe(fs.createWriteStream('./build/browser.bundle.js'));
-});
+try {
+    const feedA = JSON.parse(await sink.get('a.json'));
+    const feedB = JSON.parse(await sink.get('b.json'));
+    const bundle = await bundleJS([feedA, feedB]);
+} catch (e) {
+    // handle errors
+}
```

Comment:
Do we need to include handling errors in the docs like this? Feels a bit distracting from the actual API we are intending to explain/show.